[2024-04-16T10:36:58.566+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-16T10:36:58.632+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: priority_one_number_two_dag_v1.check_sum_number scheduled__2024-03-05T00:00:00+00:00 [queued]>
[2024-04-16T10:36:58.649+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: priority_one_number_two_dag_v1.check_sum_number scheduled__2024-03-05T00:00:00+00:00 [queued]>
[2024-04-16T10:36:58.650+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-16T10:36:58.673+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): check_sum_number> on 2024-03-05 00:00:00+00:00
[2024-04-16T10:36:58.687+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=549) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-16T10:36:58.689+0000] {standard_task_runner.py:63} INFO - Started process 553 to run task
[2024-04-16T10:36:58.689+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'priority_one_number_two_dag_v1', 'check_sum_number', 'scheduled__2024-03-05T00:00:00+00:00', '--job-id', '317', '--raw', '--subdir', 'DAGS_FOLDER/priority_one_number_two_dag.py', '--cfg-path', '/tmp/tmpiuemd96l']
[2024-04-16T10:36:58.696+0000] {standard_task_runner.py:91} INFO - Job 317: Subtask check_sum_number
[2024-04-16T10:36:58.816+0000] {task_command.py:426} INFO - Running <TaskInstance: priority_one_number_two_dag_v1.check_sum_number scheduled__2024-03-05T00:00:00+00:00 [running]> on host eaff2d74d54d
[2024-04-16T10:36:59.082+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ika' AIRFLOW_CTX_DAG_ID='priority_one_number_two_dag_v1' AIRFLOW_CTX_TASK_ID='check_sum_number' AIRFLOW_CTX_EXECUTION_DATE='2024-03-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-05T00:00:00+00:00'
[2024-04-16T10:36:59.085+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-16T10:36:59.156+0000] {logging_mixin.py:188} INFO - Total dari list number random : 627
[2024-04-16T10:36:59.157+0000] {logging_mixin.py:188} INFO - Odd Sum
[2024-04-16T10:36:59.161+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-16T10:36:59.162+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-16T10:36:59.180+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=priority_one_number_two_dag_v1, task_id=check_sum_number, execution_date=20240305T000000, start_date=20240416T103658, end_date=20240416T103659
[2024-04-16T10:36:59.240+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-16T10:36:59.386+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-16T10:36:59.390+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
